Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:20.18.0-alpine
+
+WORKDIR /app
+
+COPY yarn.lock /app
+
+COPY package.json /app
+
+RUN yarn install
+
+COPY . /app
+
+RUN yarn build
+
+EXPOSE $API_PORT
```

Comment:
it is not necessary to set the api_port from and env variable you can expose this like a hardcoded number